ivadomed/transforms.py

```diff
@@ -758,9 +758,13 @@ def undo_transform(self, sample, metadata=None):
         translation = - np.array(metadata['translation'])
 
         # Undo rotation
-        dict_params = {"rotation": [angle, axes], "scale": scale, "translation": translation}
+        dict_params = {"rotation": [angle, axes], "scale": scale, "translation": [0, 0, 0]}
+
         data_out, _ = self.__call__(sample, dict_params)
```

Comment:
Does it guarantee the rotation and scaling to be done in the right order? Or should we break it down as done with the translation?

Reply:
yes absolutely! Thank you for this catch: I updated the code. For the RandomAffine undo, the transform matrix is created in the opposite order compared when doing the transform for scaling and rotation